File: client-java/controller/src/main/java/org/evomaster/client/java/controller/mongo/operations/GreaterThanOperation.java
package org.evomaster.client.java.controller.mongo.operations;

/**
 * Represent $gt operation.
 * Selects the documents where the value of the field is greater than or equal to a specified value
 */
public class GreaterThanOperation<V> extends ComparisonOperation<V>{
    public GreaterThanOperation(String fieldName, V value) {
        super(fieldName, value);
    }
}

File: client-java/controller/src/main/java/org/evomaster/client/java/controller/mongo/operations/GreaterThanOperation.kt
package org.evomaster.client.java.controller.mongo.operations

/**
 * Represent $gt operation.
 * Selects the documents where the value of the field is greater than or equal to a specified value
 */
class GreaterThanOperation<V>(
    override val fieldName: String,
    override val value: V
): ComparisonOperation<V>(fieldName, value)